client/src/config.ts

```diff
@@ -93,8 +93,8 @@ export class Config {
 
   get debug() {
     return {
-      showParsedTokensAsWarnings: this.get<boolean>(
-        'debug.showParsedTokensAsWarnings'
+      showCollectedTokensAsWarnings: this.get<string>(
```

Comment:
What are the possible values for this option? If there are more than two, we should represent them as a string union, e.g.

```
      type ShowCollectedTokensAsWarningsOptions = 'off' | 'on' | 'something else?'
      ...
      showCollectedTokensAsWarnings: this.get<ShowCollectedTokensAsWarningsOptions>(
```

If it's just on/off, I'd prefer to keep it as a boolean

Reply:
Excellent, didn't know about string unions in TypeScipt. Much nicer.